packages/core/src/components/sections/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,76 @@
+import UINavbar from '../../navigation/Navbar'
+
+import Section from '../Section'
+
+import styles from './section.module.scss'
+
+type PageLinks = {
+  url: string
+  text: string
+}
+
+export interface NavbarProps {
+  logo: {
+    alt: string
+    src: string
+  }
+  signInButton: {
+    icon: {
+      alt: string
+      icon: string
+    }
+    label: string
+  }
+  cartIcon: {
+    alt: string
+    icon: string
+  }
+  navigation: {
+    regionalization: {
+      enabled: boolean
+      icon: {
+        alt: string
+        icon: string
+      }
+      label: string
+    }
+    pageLinks: PageLinks[]
+  }
+}
+
+function Navbar({
+  logo,
+  signInButton: {
+    label: signInButtonLabel,
+    icon: { icon: signInButtonIcon },
+  },
+  cartIcon: { icon: cartIcon },
+  navigation: {
+    regionalization: {
+      enabled: shouldDisplayRegion,
+      icon: { icon: regionIcon },
+      label: regionLabel,
+    },
+    pageLinks,
+  },
+}: NavbarProps) {
+  return (
+    <Section className={`${styles.section} section-navbar`}>
```

Comment:
What about use header for accessibility sake?
```suggestion
    <header className={`section ${styles.section} section-navbar`}>
```